### ISSUE TITLE
[GSK-1125] Overlapping UI Elements

### ISSUE THREAD

**Issue by Khushiyant** [Giskard-AI/giskard#1036]
### Issue Type

Bug

### Source

docker

### Giskard Library Version

1.9.1

### Giskard Server Version

1.5.0

### OS Platform and Distribution

macOS Ventura Version 13.4 (22F66)

### Python version

3.9

### Current Behaviour?

```shell
Current UI elements are overlapping such as:
* Download
* Refresh
* Invite
```

### Standalone code OR list down the steps to reproduce the issue

```shell
Excepted to have non-overlapping application UI elements
```

<sub>[GSK-1125](https://linear.app/giskard/issue/GSK-1125/[bug]-overlapping-ui-elements)</sub>

**Comment by Khushiyant** [comment]
<img width="1438" alt="Screenshot 2023-05-26 at 7 53 16 PM" src="https://github.com/Giskard-AI/giskard/assets/69671407/59476730-6b1f-4027-b3ca-6f7ee34d429f">

**Comment by kevinmessiaen** [comment]
This issue has been fixed with the new UI released on the 2.0 alpha version
